test/plugins/CMakeLists.txt

```diff
@@ -13,12 +13,20 @@ set (test_plugins
   EventTriggerSystem
   TestModelSystem
   TestSensorSystem
-  TestSystem
   TestWorldSystem
   MockSystem
   Null
 )
 
+# TODO: someone with knowledge of ign-plugin please resolve:
+# TestSystem.obj : error LNK2001: unresolved external symbol IgnitionPluginHook
```

Comment:
This looks like the visibility problem I was having in https://github.com/ignitionrobotics/ign-rendering/pull/180, it may be a visibility flag needing to be fixed in `ign-plugin`. I think leaving this note here is fine for now.